packages/editor/src/functions/utils.ts

```diff
@@ -34,8 +34,23 @@ export function unique(arr, maybeComp) {
   }
   return newArr
 }
-export const isApple = /(Mac|iPhone|iPod|iPad)/i.test((process as any).browser && navigator ? navigator.platform : '')
-export const cmdOrCtrlString = isApple ? '⌘' : 'ctrl'
+
+export const isApple = () => {
+  const iOS_1to12 = /iPad|iPhone|iPod/.test(navigator.platform)
+
+  const iOS13_iPad = navigator.platform === 'MacIntel'
+
+  const iOS1to12quirk = () => {
+    var audio = new Audio() // temporary Audio object
+    audio.volume = 0.5 // has no effect on iOS <= 12
+    return audio.volume === 1
+  }
+
+  return iOS_1to12 || iOS13_iPad || iOS1to12quirk()
```

Comment:
Can you explain why this is necessary?

Reply:
I have the latest mac and it kept showing ctrl + s for it while it should show cmd + s.

The reason behind this is that for latest macs it return `MacIntel` as navigator.platform which wasn't being caught in the previous solution.

So i found an interesting solution on stackoverflow for this which i implemented above.

Reply:
Does this solution also work with older Macs?